src/main/kotlin/com/example/olaf/blogtutorial/Entities.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.olaf.blogtutorial
+
+import java.time.LocalDateTime
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+
+@Entity
+class Article(
+    var title: String,
+    var headline: String,
+    var content: String,
+    @ManyToOne var author: User,
+    var slug: String = title.toSlug(),
+    var addedAt: LocalDateTime = LocalDateTime.now(),
+    @Id @GeneratedValue var id: Long? = null
```

Comment:
보통 Database에서 id값은 가장 앞에 있으니 가장 위로 올리는것도 좋을것 같아요
그리고 Id 값은 고유값이 primary key 될텐데 null을 허용하는건 아니될거 같아요

```suggestion
    @Id @GeneratedValue 
    val id: Long = 0L
```
생성자에서 제외하고 내부에 선언하면 좋을것 같아요~

Reply:
추가로 
var (변수)
val (상수)
의 차이를 잘 이해하시구 사용하시면 더 좋을 것 같아요 :)
id 값은 보통 한번 생성되고나서는 변경되지 않는 불변의 값이니 
val이 더 맞겠지요~

Reply:
ㅎㅎ넴 앞으로는 이렇게 쓰겠습니다